langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/googleai/PartsAndContentsMapper.java

```diff
@@ -182,7 +182,7 @@ static AiMessage fromGPartsToAiMessage(List<GeminiPart> parts, boolean includeCo
 
             String text = part.getText();
             if (text != null && !text.isEmpty()) {
-                fullText.append(text);
+                fullText.append("\n\n").append(text);
```

Comment:
Shouldn't `\n\n` be appended only when `fullText` already contains something?

Reply:
Correct, fixed!